GCI 2019 task TrungNT (#30)
HaikuArchives/Tipster: tips/tips-en.txt

> gtTipster

Reviewer: ```suggestion
Tipster
```

Hold SHIFT while scrolling with the mouse wheel to do page-wise scrolling in Tracker windows and drill-down context menus.
https://www.haiku-os.org/docs/userguide/en/tracker.html
%
GUI

> You can quickly minimize an active window by using CTRL + ALT + M, and minimize all windows of that application by using CTRL + ALT + H.

Reviewer: A bit shorter and without the repetition:
```suggestion
Quickly minimize an active window by using CTRL + ALT + M, and minimize all windows of that application with CTRL + ALT + H.
```

https://www.haiku-os.org/docs/userguide/en/keyboard-shortcuts.html
%
Miscellaneous

> Quickly switching to another workspace by pressing ALT + Fx (with Fx being the function key corresponding to workspace number X), add SHIFT to bring the active window with you.

Reviewer: A bit shorter:
```suggestion
Quickly switch to another workspace by pressing ALT + Fx (Fx being the function key corresponding to workspace number X), add SHIFT to take the active window with you.
```

https://www.haiku-os.org/docs/userguide/en/keyboard-shortcuts.html
%
Preferences

> The Deskbar can be moved to all corners as well as the top or bottom of the screen, and also folded to a more compact layout by gripping and dragging the knobby area on the tray. 

Reviewer: I think we need two sentences to be clear, and be specific about the knobby widget:

```suggestion
The Deskbar can be moved to all corners as well as the top or bottom of the screen by grabbing the knobbly widget beside the clock. It can be folded to a more compact layout by dragging the widget onto the tray. 
```

https://www.haiku-os.org/docs/userguide/en/deskbar.html
%
Preferences

> The Deskbar preferences allows you to change the icon size, auto-hide the taskbar, hide application names, and a lot more.

Reviewer: ```suggestion
The Deskbar preferences allow you to change icon size, auto-hide the Deskbar, hide application names, and more.
```

https://www.haiku-os.org/docs/userguide/en/deskbar.html
%
Terminal

> In addition to the UNIX ps/kill combo, Haiku lets you kill an app by using kill appname.

Reviewer: ```suggestion
In addition to using the UNIX ps/kill combo with an app's process ID, Haiku lets you kill an app with 'kill appname'.
```
